admin/acceptance.py

```diff
@@ -554,11 +552,15 @@ def configured_cluster_for_nodes(
         dataset_backend_config_file=dataset_backend_config_file
     )
 
-    configuring = perform(
-        make_dispatcher(reactor),
-        configure_cluster(
+    commands = Effect(Func(
```

Comment:
:question: This is different from how we use Effect elsewhere in the provisioning code. Trying to think through if that will cause us any troubles.

- Do we still generate our docs using Effect? Will this break that?
- Are there any meaningful unit tests that we write or want to write against our provisioning code that this will slow down?

The answer to the first bullet should be answered by Jenkins, and I think the answer to the second is a tentative no, so should be fine just to do inline like this. If you think the answer to the second is actually yes, please refactor into a new proper intent rather than just using Func.